GameOfLifeConsole/Game.cs

```diff
@@ -0,0 +1,103 @@
+﻿using GameOfLifeConsole.Services;
+
+namespace GameOfLifeConsole
+{
+    public class Game
```

Comment:
Please add comments for class and methods.

Reply:
Done. Thank you.